modules/virt-enabling-mac-address-pool-for-namespace-cli.adoc

```diff
@@ -5,15 +5,15 @@
 [id="virt-enabling-mac-address-pool-for-namespace-cli_{context}"]
 = Enabling a MAC address pool for a namespace in the CLI
 
-Enable a MAC address pool for virtual machines in a namespace by applying the `mutatevirtualmachines.kubemacpool.io=allocate` label to the namespace.
+If you disabled KubeMacPool for a namespace and want to re-enable it, remove the `mutatevirtualmachines.kubemacpool.io=ignore` label from the namespace.
```

Comment:
Since upgrade cluster could still have `allocate`, I think that it would be worthwhile to add 
```suggestion
If you disabled KubeMacPool for a namespace and want to re-enable it, remove the `mutatevirtualmachines.kubemacpool.io=ignore` label from the namespace. Former versions suggested to set `mutatevirtualmachines.kubemacpool.io=allocate`. This is still supported but redundant.
```

Reply:
Good point. I've modified this a bit and added it as a note to the intro para:
`[NOTE]
====
Earlier versions of {VirtProductName} used the label `mutatevirtualmachines.kubemacpool.io=allocate` to enable KubeMacPool for a namespace. This is still supported but redundant as KubeMacPool is now enabled by default.
====`